app-engine/java/src/com/google/cloud/tools/intellij/appengine/java/facet/standard/AppEngineStandardSupportProvider.java

```diff
@@ -71,6 +79,14 @@
   private static final Logger LOG =
       Logger.getInstance("#com.intellij.appengine.facet.AppEngineStandardSupportProvider");
 
+  private static final NotificationGroup NOTIFICATION_GROUP =
+      new NotificationGroup(
+          new PropertiesFileFlagReader().getFlagString("notifications.plugin.groupdisplayid"),
+          NotificationDisplayType.BALLOON.BALLOON,
+          true,
+          null,
+          GoogleCloudCoreIcons.CLOUD);
```

Comment:
regarding Patrick's comment on error style for this message, can we consider having Google Cloud icon in "error state", i.e. something with a red asterisk or similar? If it takes long to obtain a new icon, we can postpone for later.